packages/storybook/stories/va-telephone.stories.jsx

```diff
@@ -9,32 +9,70 @@ import {
   PRIMARY_STORY,
 } from '@storybook/addon-docs/blocks';
 
-import {
-  CONTACTS,
-  contactsMap,
-  Table,
-} from '@department-of-veterans-affairs/component-library';
-
 import { getWebComponentDocs, propStructure } from './wc-helpers';
 
 const telephoneDocs = getWebComponentDocs('va-telephone');
-//
-// This builds the available "CONTACTS" list table
-// Descriptions are available in the contacts.js file
-const fields = [
-  { label: 'Property name (CONTACTS.x)', value: 'key' },
-  { label: 'Phone number', value: 'value' },
-  { label: 'Description', value: 'description' },
+
+const columns = ['Property name (CONTACTS.x)', 'Phone number', 'Description'];
+const rows = [
+  ['222_VETS', '8772228387', 'VA Help Line'],
+  ['4AID_VET', '8774243838', 'National Call Center for Homeless Veterans'],
+  ['711', '711', 'Telecommunications Relay Service'],
+  ['911', '911', '911'],
+  ['CAREGIVER', '8552603274', 'VA National Caregiver Support Line'],
+  ['CRISIS_LINE', '8002738255', 'Veterans Crisis hotline'],
+  ['CRISIS_TTY', '8007994889', 'Veterans Crisis hotline TTY'],
+  ['DMC', '8008270648', 'Debt Management Center'],
+  ['DMC_OVERSEAS', '6127136415', 'Debt Management Center (Overseas)'],
+  [
+    'DMDC_DEERS',
+    '8663632883',
+    'Defense Manpower Data Center (DMDC) | Defense Enrollment Eligibility Reporting System (DEERS) Support Office',
+  ],
+  ['DS_LOGON', '8005389552', 'Defense Manpower Data Center'],
+  ['DS_LOGON_TTY', '8663632883', 'Defense Manpower Data Center TTY'],
+  ['FEDERAL_RELAY_SERVICE', '8008778339', 'Federal Relay Service'],
+  ['GI_BILL', '8884424551', 'Education Call Center (1-888-GI-BILL-1)'],
+  ['GO_DIRECT', '8003331795', 'Go Direct/Direct Express (Treasury)'],
+  ['HELP_DESK', '8006982411', 'VA Help desk'],
+  [
+    'HEALTHCARE_ELIGIBILITY_CENTER',
+    '8554888440',
+    'VA Healthcare Eligibility Center (Eligibility Division)',
+  ],
+  ['HELP_TTY', '8008778339', 'VA Help Desk TTY'],
+  ['MY_HEALTHEVET', '8773270022', 'My HealtheVet help desk'],
+  ['NCA', '8005351117', 'National Cemetery Scheduling Office'],
+  ['SUICIDE_PREVENTION_LIFELINE', '8007994889', 'Suicide Prevention Line'],
+  ['TESC', '8882242950', 'U.S. Treasury Electronic Payment Solution Center'],
+  [
+    'TREASURY_DMS',
+    '8888263127',
+    'U.S. Department of the Treasury (Debt Management Services)',
+  ],
+  ['VA_311', '8006982411', 'VA Help desk (VA411)'],
+  ['VA_411', '8006982411', 'VA Help desk (VA411)'],
+  ['VA_BENEFITS', '8008271000', 'Veterans Benefits Assistance'],
 ];
 const Contacts = () => (
-  <Table
-    fields={fields}
-    data={Object.entries(contactsMap).map(c => ({
-      key: c[0],
-      value: <va-telephone contact={c[1].phoneNumber} />,
-      description: <div style={{ maxWidth: '30em' }}>{c[1].description}</div>,
-    }))}
-  />
+  <va-table>
```

Comment:
Converted to use `va-table` instead of the other React Component

Reply:
I know this is just in our storybook, but should we be using this before work is done? @bkjohnson

Reply:
I'd like to have this work be separate. I had planned to write a ticket [for this](https://dsva.slack.com/archives/CBU0KDSB1/p1645220464229119?thread_ts=1645202969.999719&cid=CBU0KDSB1) which would hopefully result in the `CONTACTS` object being exported from the `web-components` sub-package, so I'd also like to not redefine all of that in here just for the table.

Reply:
@bkjohnson @k80bowman sounds good I just reverted. For future reference when that update is made here is the commit: https://github.com/department-of-veterans-affairs/component-library/pull/313/commits/06c4ca348295d8689d1b69c8ddb738ada132b8e6